src/main.cpp

```diff
@@ -107,8 +112,30 @@ bool download(DownloadEnum::Category cat, const char* name)
 	return true;
 }
 
+#ifdef _WIN32
+
+static std::vector<std::string> argv_strings;
+static std::vector<const char*> argv_data;
+
+void prepare_args(int *argc, char*** argv) {
+	wchar_t** argv_w = CommandLineToArgvW(GetCommandLineW(), argc);
```

Comment:
Keep this in mind:

> CommandLineToArgvW allocates a block of contiguous memory for pointers to the argument strings, and for the argument strings themselves; the calling application must free the memory used by the argument list when it is no longer needed. To free the memory, use a single call to the [LocalFree](https://docs.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-localfree) function.

-- https://docs.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-commandlinetoargvw

Reply:
Ah, good catch, I've missed that, thank you!